gslib/commands/mb.py

```diff
@@ -243,7 +247,9 @@ def RunCommand(self):
     rpo = None
     if self.sub_opts:
       for o, a in self.sub_opts:
-        if o == '-l':
+        if o == '--autoclass':
```

Comment:
I am guessing this feature is only expected in json API. In that case, make sure to raise an error if this gets tried with XML API. I have rolled out this PR https://github.com/GoogleCloudPlatform/gsutil/pull/1358 to consolidate this error check in one place. You can rebase once the PR gets submitted and adding the flag to json_only_flags.

Reply:
Done!